lib/cloud/aws/policy_statements.go

```diff
@@ -137,6 +139,24 @@ func StatementForEC2InstanceConnectEndpoint() *Statement {
 	}
 }
 
+// StatementForAWSAppAccess returns the statement that allows AWS App Access.
+// Only IAM Roles with `teleport.dev/integration: Allowed` Tag can be used.
+func StatementForAWSAppAccess() *Statement {
+	requiredTag := types.TeleportNamespace + "/integration"
+	return &Statement{
+		Effect: EffectAllow,
+		Actions: []string{
+			"sts:AssumeRole",
+		},
+		Resources: allResources,
+		Conditions: map[string]map[string]SliceOrString{
+			"StringEquals": {
+				"iam:ResourceTag/" + requiredTag: SliceOrString{"Allowed"},
```

Comment:
nit: for consistency, I would use lowercase for values. Or, I believe we use `true` as the value in most places to toggle/enable something,  like `teleport.dev/managed=true`, `teleport.dev/insecure-skip-verify=true` , `teleport.dev/ignore=true`

Reply:
Got it :+1: Changed